docs/substrate-based-crab-chain/crab-tut-create-account.md

```diff
@@ -5,102 +5,38 @@ sidebar_label: Create an Account
 
-## 1. Crab Web Wallet
+> NOTE: Not all old accounts need to be migrated, and accounts that have been imported into the polkadot{.js} extension do not need to be migrated, meanwhile, these accounts will not be displayed in the `Account Migration`.
+> 
 
-![setting](../assets/substrate-based-crab-chain/create-account-01.png)
+## 1. How to Connect Polkadot{.js} Extension to Darwinia/Crab
 
-Enter [Crab Web Wallet](https://apps.darwinia.network), you can see two buttons "Add Account" and "Restore JSON" in the "Account" column.
+This part will walk you through the process of connecting your [polkadot{.js} extension](https://polkadot.js.org/extension/) to Darwinia.
 
-![create account](../assets/substrate-based-crab-chain/create-account-02.png)
+### Install the Polkadot{.js} Extension
 
-**New account**
+If you do not already have the polkadot{.js} extension installed, you can install the extension from the [Chrome store](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd). For help with installation, you can follow the Getting Started with the polkadot{.js} extension guide from the official polkadot{.js} extension documentation. Once you have installed the polkadot{.js} extension, follow the account creation wizard. Make sure to store your mnemonic safely and do not share it with anyone. If you prefer, there is also a video tutorial that you can check out [here.](https://www.youtube.com/watch?v=sy7lvAqyzkY)
 
-Click "Add Account", after setting the basic account information, click the "Save" button. (By default, only "mnemonic", "private key" need to be switched)
+When the polkadot{.js} extension is fully set up, you’ll have your address displayed at the top of the extension window.
 
-![create account](../assets/substrate-based-crab-chain/create-account-03.png)
+### Connect to Darwinia Apps
 
-Click the "Create and Backup Account" button to back up the account "json file"
+Once you have an account, you can enter [Apps](https://apps.darwinia.network/#/account) and connect to Darwinia by clicking `Connect Wallet` on the right corner of the top, and you also can switch the network connected by clicking and choosing the network through the button on the left upper.
 
-![create account](../assets/substrate-based-crab-chain/create-account-04.png)
+## 2. Account Migration
 
-> Be sure to back up `mnemonics, private keys, json files`, etc. When backing up the json file, please keep the password safe. If the password is lost, the address cannot be restored through the json file, but it can be re-imported through the mnemonic word and private key.
+If your accounts are in the old version and cannot be found in your [polkadot{.js} extension](https://polkadot.js.org/extension/), you can restore account JSON through `Account Migration` and add the JSON to polkadot{.js} extension.
```

Comment:
I'd like to know when the Polkadot JS extension can't find an account.

Reply:
If I create an account in the old version Apps, and have never import JSON to polkadot js extension.